proposals/description/fip_82b.ts

```diff
@@ -0,0 +1,463 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_82b: ProposalDescription = {
+  title: 'FIP-82b: Authorise the TribalCouncil with necessary roles',
+  commands: [
+    /////// Transfer all non-major role admins to the ROLE_ADMIN, to allow TribalCouncil to manage ////////
+    {
+      target: 'core',
+      values: '0',
+      method: 'createRole(bytes32,bytes32)',
+      arguments: [
+        '0xf0b50f04623eeaacfa1f202e062a3001c925a35c6b75d6903e67b43f44bbf152', // PARAMETER_ADMIN
```

Comment:
PARAMETER_ADMIN doesn't appear anywhere in PCV. or the permissions test. I prefer removing it

Reply:
It's used in the `NonCustodialPSM`, but hasn't been created or granted out yet. We should leave as it's in an access modifier on `NonCustodialPSM`

Reply:
imo the roles should be consolidated/clarified. There is a PSM_ADMIN_ROLE and a MINOR_PARAMETER_ADMIN. At least one of those should replace this one or vice versa

Reply:
As discussed, leaving for now